src/Backend/Core/Engine/Base/Action.php

```diff
@@ -96,15 +95,15 @@ public function getAction(): string
      */
     public function checkToken(): void
     {
-        $fromSession = (\SpoonSession::exists('csrf_token')) ? \SpoonSession::get('csrf_token') : '';
-        $fromGet = $this->getContainer()->get('request')->query->get('token');
+        $fromSession = BackendModel::getSession()->get('csrf_token', '');
```

Comment:
No. But if no session can be retrieved than the CSRF-check can't be valid. 
But maybe I don't understand the current implementation, and the session isn't started on the correct places, and therefore needs to be started in this method.